### ISSUE TITLE
load_photo results in daemon crash

### ISSUE THREAD

**Issue by goforthtech** [vysheng/tg#433]
When trying to execute 'load_photo', telegram-cli is crashing.  

Setup:
Fedora 20 - repository version
Running telegram-cli as a daemon:
     /bin/telegram-cli -E -N -d -P 10234 -I

telnet to localhost 10234
execute 'dialog_list'
history user#123456789
load_photo [msg id]
Result: remote session disconnected

ps aux shows that telegram-cli has exited.

Check download directory, and the file IS there (default directory of ~/.telegram-cli/downloads) but failed to report the path and exited.

If I re-launch it, then Telnet back into the session and try execute load_photo again with the SAME message number, it reports back correctly and continues just fine.

**Comment by alesantosp** [comment]
I had the same error, found a solution?

**Comment by goforthtech** [comment]
Nope.  Never did..  When I run the app in full verbose (-vvv) it doesn't reveal anything either.  It just exits.
